src/index.js

```diff
@@ -1,12 +1,36 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
+import { createStore } from 'redux';
+import rootReducer from './reducers/index';
 import './index.css';
-import App from './App';
+import App from './components/App';
 import reportWebVitals from './reportWebVitals';
 
+const initialState = {
+  books: {
+    [Math.floor((Math.random() * 100))]: {
+      title: 'YDKJS',
+      category: 'Learning',
+    },
+    [Math.floor((Math.random() * 100))]: {
+      title: 'Star Wars',
+      category: 'Sci-fi',
+    },
+    [Math.floor((Math.random() * 100))]: {
+      title: 'Dunkirk',
+      category: 'History',
```

Comment:
### [Optional]
Kindly be aware that you can use uuidv4 to automatically generate ids. read more 👉 [npm uuid](https://www.npmjs.com/package/uuidv4)

Example:
```
import uuid from 'uuidv4'
id: uuid(): {title: 'YDKJS', category: 'Learning'}
```